src/lib/pickles/pickles.ml

```diff
@@ -439,6 +552,8 @@ module Make_str (_ : Wire_types.Concrete) = struct
              * Cache.Wrap.Key.Verification.t
         -> ?return_early_digest_exception:bool
         -> ?override_wrap_domain:Pickles_base.Proofs_verified.t
+        -> ?override_wrap_main:
+             (max_proofs_verified, branches, prev_varss) wrap_main_generic
```

Comment:
possible typo with `varss`

Reply:
The name is because it's a list of `vars`. Improved name suggestions are welcome (although not for me to tweak in this PR!)